catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -266,6 +273,15 @@ private SchematronReport generateReport(String metadata, Templates validator)
         return report;
     }
 
+    /**
+     * Replace tabs, literal carriage returns, and newlines with a single whitespace
+     * @param input
+     * @return
+     */
+    static String sanitize(final String input) {
```

Comment:
I am a fan of testing the public contract of a class to avoid over coupling the tests to the current implementation.  Increasing the visibility of methods for testing is a code smell to me.